src/Core/PostgreSQLProtocol.cpp

```diff
@@ -28,6 +28,8 @@ ColumnTypeSpec convertTypeIndexToPostgresColumnTypeSpec(TypeIndex type_index)
             return {ColumnType::FLOAT8, 8};
 
         case TypeIndex::FixedString:
+            return {ColumnType::BPCHAR, -1};
```

Comment:
This is not strictly needed for this particular PR, but I thought we'd rather have the proper `ColumnType` anyways